capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -50,17 +50,17 @@ export class WikiMediaComponent implements OnInit, OnDestroy {
       let endIndex = firstPartOfString.indexOf("<h2>", 0);
       let startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
-      let finalString = middleOfString.split('href="/wiki').join('target="_blank" href="https://en.wikipedia.org/wiki');
-      finalString.split('title="Edit section: History">edit').join('>');
-      return finalString;
+      let paragraphs = middleOfString.split('<p>');
+      if (paragraphs.length > 6){
+        middleOfString = paragraphs.slice(0,5).join('');
+      }
+    //  text = middleOfString.split(/<a.*?\/a>/g).join("");
+      let history = middleOfString.split(/<.*?>/g).join("");
+      return history.split(/&.*?;/g).join("");
     }
     console.log("The city page was found, but unfortunately there was no history paragraph found!");
```

Comment:
Does this happen often? Do you think we should show the first paragraph or so of the regular article in this case as it probably includes history info of when it was founded?